Resources/Prototypes/_Goobstation/Changeling/radio_channels.yml

```
@@ -0,0 +1,7 @@
+- type: radioChannel
+  id: Hivemind
+  name: chat-radio-hivemind
+  keycode: 'g'
+  frequency: 19840
+  color: "#e36b00"
+  longRange: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)